versions.properties

```diff
@@ -66,8 +68,8 @@ org.apache.logging.log4j:log4j-flume-ng=$log4jVersion
 org.apache.logging.log4j:log4j-iostreams=$log4jVersion
 org.apache.logging.log4j:log4j-jul=$log4jVersion
 org.apache.logging.log4j:log4j-slf4j-impl=$log4jVersion
-httpClientVersion=4.5.3
-httpCoreVersion=4.4.6
+httpClientVersion=4.5.6
+httpCoreVersion=4.4.10
```

Comment:
ugh, they are different? that makes me sad

Reply:
They are. It makes me sad, too. I checked multiple times, though, and I am certain that I matched the versions correctly.